index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="connect" href="InaiMathi.ttf">
+  <link rel="preconnect" href="https://fonts.gstatic.com"> 
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+    type="text/css"
+  />
+  <link href="styles.css" rel="stylesheet" />
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <a href="#" class="log">Kaneez Fatima</a>
+      <a><i class="fa fa-bars cursor"></i></a>
+    </nav>
+  </header>
+      <section class="main-section img">
+        <div>
+          <h1><b>Hey There.</b></h1>
+            <h1><b>I'm Kaneez Fatima</b></h1>
+            <h2>I'am a Software Developer</h2>
```

Comment:
- [ ] [OPTIONAL] Size up the "Hey There" with font-size: 54px, as the original design